### PR TITLE
Releases watchmaker version 0.21.9

### PR DESCRIPTION
**Pull request by eemperor**

### CODE REVIEW

**Comment by eemperor** [comment]
go codebuild go

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 0.21.9
+
+**Commit Delta**: [Change from 0.21.8 release](https://github.com/plus3it/watchmaker/compare/0.21.8...0.21.9)
+
+**Released**: 2021.04.26
+
+**Summary**:
+
```
Comment (lorengordon) [inline CHANGELOG.md:10]: Can we add a top-level bullet here about supporting Salt 3003? I'm also working on a PR to add a blurb about the salt support to the docs...

Comment (eemperor) [reply CHANGELOG.md:10]: Done

**Comment by eemperor** [comment]
go codebuild go

**Review by lorengordon — APPROVED** [review]
:shipit:

**Pull request merged**